renderer/src/components/ChartsDashboard.jsx

```
@@ -0,0 +1,251 @@
+      <div className="mb-8 text-center">
+        <h1 className="text-3xl font-bold text-gray-800 mb-2">Charts Dashboard</h1>
+        <p className="text-gray-600">
+          Generate and send various charts to Discord channels
+        </p>
+      </div>
+
+      {/* Feature status indicators */}
+      <div className="grid grid-cols-1 md:grid-cols-2 gap-4 mb-6">
+        <div className={`p-4 rounded-lg ${features.mapToolsEnabled ? 'bg-green-50 border border-green-200' : 'bg-gray-100 border border-gray-200'}`}>
+          <div className="flex items-center">
+            <div className={`w-3 h-3 rounded-full mr-2 ${features.mapToolsEnabled ? 'bg-green-500' : 'bg-gray-400'}`}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)